### PR TITLE
Adds ability to see info about a table column.

### LINKED ISSUE #1992: Ability to see info about a Table column
Part of https://github.com/bigbinary/neeto-engineering-ui-ux/issues/211

@deepakjosp _a

### PR DESCRIPTION
**Pull request by deepakjosp**
- Fixes #1992.

**Description**

**Checklist**

- [x] I have made corresponding changes to the documentation.
- [x] I have updated the types definition of modified exports.
- [x] I have verified the functionality in some of the neeto web-apps.
- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

### CODE REVIEW

**Comment by deepakjosp** [comment]
@praveen-murali-ind  _a could you please work on the UI.

**Comment by praveen-murali-ind** [comment]
@deepakjosp _a Please work on the changes discussed.

**Comment by praveen-murali-ind** [comment]
@ajmaln _a Could you please review.

**Review by ajmaln — COMMENTED** [review]
To fix: The popover is also triggering drag and drop of columns when it is draggable.

**Comment by praveen-murali-ind** [comment]
@anil-anand _A Could you please work on the review suggestions as Deepak is on PTO?

File: src/components/Table/hooks/useColumns.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  const handleResize =
+    index =>
+    (_, { size }) => {
+      const nextColumns = [...columns];
+      nextColumns[index] = { ...nextColumns[index], width: size.width };
+      setColumns(nextColumns);
+    };
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```
[the remark this answers is not in the document]
Comment (anil-anand) [reply src/components/Table/hooks/useColumns.js]: Looks like it's not. I have removed the comment.

**Comment by anil-anand** [comment]
@ajmaln _a
I have made the suggested changes. Please watch [this video](https://anil-anand.neetorecord.com/watch/f0fd3004-080e-4dc2-afd5-2da56aea10ba) and review the changes.

**Review by ajmaln — APPROVED** [review]
@anil-anand _a Changes looks good. Please fix the eslint errors and also update the PR checklist and assign back

**Comment by deepakjosp** [comment]
@deepakjosp  _a

**Comment by deepakjosp** [comment]
@ajmaln _a changes are done. please review.

**Review by ajmaln — APPROVED** [review]

**Pull request merged**